src/components/templates/ScreenShare/components/Player/Player.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useRef, useEffect } from "react";
+import classNames from "classnames";
+import {
+  ILocalVideoTrack,
+  IRemoteVideoTrack,
+  ILocalAudioTrack,
+  IRemoteAudioTrack,
+} from "agora-rtc-sdk-ng";
+import "./Player.scss";
+
+import VolumeMutedIcon from "assets/icons/volume-muted-icon.svg";
+import CameraOffIcon from "assets/icons/camera-off-icon.svg";
+import StreamingIcon from "assets/icons/streaming-icon.svg";
```

Comment:
As described above, we should reuse the icons from `@fortawesome`

Reply:
Aligned to what we are already using in the generic twilio video components.